setup.py

```diff
@@ -15,7 +15,8 @@
         'gym',
         'pybullet',
         'stable_baselines3',
-        'ray[rllib]'
+        'ray[rllib]',
+        'tensorboard'
```

Comment:
Did this fix the issue for you?
Could I kindly ask, what are your machine specifications? As mine are as follows

`Darwin jail-master 21.1.0 Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:23 PDT 2021; root:xnu-8019.41.5~1/RELEASE_X86_64 x86_64`

And unfortunately adding tensorboard does not fully aid the issue for me. In particular, the script `examples/learn.py` keeps failing on ray imports.

The following is the end of error log:
`ImportError: ray.tune in ray > 0.7.5 requires 'tabulate'. Please re-run 'pip install ray[tune]' or 'pip install ray[rllib]'.`

Reply:
This fixes it on my personal machine running 18.04 Ubuntu. I noticed the raylib import error on the pypi publish workflow I'm working on including in this PR which runs on Ubuntu 20.04. 

Will comment here when fixed.

Reply:
Thank you! 
I tested in a virtual machine run with ubuntu bionic 18.04 x86_64 (similar to yours), and it caused the same error. It seems that the setup.py script is failing to properly resolve the dependencies for fresh conda environment.

Simply running
`pip install ray[rllib] ray[tune]` fixes the issue on both machines.

Cheers.

Reply:
Great to hear!

I believe that `setup.py` is not needed anymore according to [pip docs](https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/).
I have forked the project earlier, and removed `setup.py` completely, and it works just fine ([commit](https://github.com/hidal00p/gym-pybullet-drones/commit/3635b132bfedb2c9d4e4c4aaf841b9b8abc72ca5)).

Reply:
You're completely right! Removed setup.py